app/controllers/bulk_actions/catalog_record_id_and_barcode_jobs_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module BulkActions
+  class CatalogRecordIdAndBarcodeJobsController < ApplicationController
+    include CreatesBulkActions
+    self.action_type = "SetCatalogRecordIdsAndBarcodesJob"
+
+    def job_params
+      super.merge(use_catalog_record_ids_option: params[:use_catalog_record_ids_option],
+        catalog_record_ids: params[:catalog_record_ids],
+        use_barcodes_option: params[:use_barcodes_option],
+        barcodes: params[:barcodes])
+    end
+  end
+end
```

Comment:
This was a `git mv` followed by `s/catkey/catalog_record_id/`, so it looks like a new file but it really isn't.